__std__/flake.nix

```diff
@@ -0,0 +1,248 @@
+# The flake.nix is the entrypoint of all nix code.
+#
+# This repository uses the std tool https://github.com/divnix/std.
+# Familiarity with std is required to be able to contribute effectively.
+# While official documentation for std can be found in its GitHub, this flake
+# has been thoroughly commented so as to quickstart new maintainers.
+#
+# Most of what there is to know about the nix code inside this repository
+# can be learned by reading this file. A second read will be needed as some
+# std-specific terms may be referenced first and defined later.
+{
+  description = "Plutus Core - The Scripting Language for Cardano";
+
+  # The flake inputs will be accessible by name in each nix file like so:
+  # { inputs, cell }: inputs.nixpkgs, inputs.haskell-nix
+  inputs = {
+    nixpkgs = {
+      url = "github:NixOS/nixpkgs/34e4df55664c24df350f59adba8c7a042dece61e";
+    };
+    std = {
+      url = "github:divnix/std";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+    haskell-nix = {
+      url = "github:input-output-hk/haskell.nix";
+      inputs = {
+        hackage.follows = "hackage-nix";
+      };
+    };
+    hackage-nix = {
+      url = "github:input-output-hk/hackage.nix";
+      flake = false;
+    };
+    sphinxcontrib-haddock = {
+      url = "github:michaelpj/sphinxcontrib-haddock";
+      flake = false;
+    };
+    # cardano-repo-tool = {
+    #   url = "github:input-output-hk/cardano-repo-tool";
+    #   flake = false;
+    # };
+    # gitignore-nix = {
+    #   url = "github:hercules-ci/gitignore.nix";
+    #   flake = false;
+    # };
+    # haskell-language-server = {
+    #   # Pinned to a release
+    #   url = "github:haskell/haskell-language-server?ref=1.7.0.0";
+    #   flake = false;
+    # };
+    # iohk-nix = {
+    #   url = "github:input-output-hk/iohk-nix";
+    #   flake = false;
+    # };
+    # pre-commit-hooks-nix = {
```

Comment:
It might be better to use `lefthook` that already ships as a Nixago pebble in `std`.

It' way faster and has way better nix-UX (IMO).

See: https://divnix.github.io/std/reference/std/nixago/lefthook.html

Reply:
I'll try it out and see how expensive it is to migrate from pre-commit-hooks-nix 👍